src/types/global.d.ts

```diff
@@ -28,15 +28,25 @@ declare global {
       {
         metadataFilter?: { key: string; value?: string };
         filterVisible?: boolean;
+        ids?: string[];
       },
       { subjects: GQLSubject[] }
     >;
-    subjectLoader: DataLoader<
+    nodeLoader: DataLoader<
       {
         id?: string;
       },
       Node
     >;
+    nodesLoader: DataLoader<
+      {
+        contextId?: string;
+        metadataFilter?: { key: string; value?: string };
+        filterVisible?: boolean;
+        ids?: string[];
+      },
```

Comment:
Føler vi burde ha en type for dette. Det blir vel duplisert et annet sted også.

Reply:
Det er delvis overlappende med en anna type, så kan sjå om eg finn på nokke lurt med det.